projects/plugins/jetpack/modules/contact-form/grunion-response-email-template.php

```diff
@@ -0,0 +1,277 @@
+
+										<a class="build-your-own-jetpack-new-section-link" href="https://jetpack.com/support/jetpack-blocks/contact-form/#export-form-responses" style="background-color: #000; border-radius: 4px; display: inline-block; box-sizing: border-box; color: #fff; font-weight: 600; letter-spacing: -0.02em;line-height: 24px; padding: 12px 24px; text-decoration: none; font-size: 16px;">' . esc_html__( 'Learn more', 'jetpack' ) . '</a>
+									</td>
+								</tr>
+								<tr>
+									<td class="footer-november-2022" style="border-collapse: collapse; font-family: \'SF Pro Display\', Helvetica, Arial, sans-serif; padding: 48px 56px;">
+										<table style="border-collapse: collapse; width: 100%%;">
+											<tr>
+												<td style="border-collapse: collapse;">
+													<img src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/jetpack-icon.png" class="footer-november-2022-jetpack-icon" width="20" height="20" style="border: 0 none; height: auto; line-height: 100%%; outline: none; text-decoration: none; display: inline-block; margin-bottom: 16px;">
+												</td>
+											</tr>
+											<tr>
+												<td align="left" style="border-collapse: collapse;">
+												<h2 class="footer-november-2022-header-text" style="color: #101517; font-size: 16px; line-height: 1.2; margin-bottom: 8px; margin-top: 0;">' . esc_html__( 'Get Jetpack on the go', 'jetpack' ) . '</h2>
+
+													<p class="footer-november-2022-body-text" style="direction: ltr; color: #3c434a; font-size: 14px; line-height: 1.4; margin-bottom: 16px; margin-top: 0;">' . esc_html__( 'View site activity and stats, get notifications when your site is down, fix malware threats, and restore your site from anywhere.', 'jetpack' ) . '</p>
+												</td>
+											</tr>
+											<tr>
+												<td align="left" class="footer-november-2022-app-links" style="border-collapse: collapse;">
+													<a href="https://play.google.com/store/apps/details?id=com.jetpack.android" style="color: #00AADC; text-decoration: none; width: 115px; display: inline-block; margin-bottom: 64px;"><img src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/play-store-badge.png" class="footer-november-2022-app-image" width="115" height="33" style="line-height: 100%%;outline: none; text-decoration: none; border: 0 none; display: inline-block; height: 33px;"></a>
+
+													<a href="https://apps.apple.com/us/app/jetpack-website-builder/id1565481562" style="color: #00AADC; text-decoration: none; width: 115px; display: inline-block; margin-bottom: 64px;"><img src="https://s0.wp.com/wp-content/mu-plugins/html-emails/themes/jetpack/images/app-store-badge.png" class="footer-november-2022-app-image" width="115" height="33" style="line-height: 100%%; outline: none; text-decoration: none; border: 0 none; display: inline-block; height: 33px;"></a>
```

Comment:
Does loading images from a wp.com subdomain have any positive or negative impact on the deliverability of the emails?
Given that the contact form feature can be used by folks who did not connect their site to WordPress.com, and who did not agree to our ToS, maybe we should load those images from each site instead?

Reply:
I'm not sure if loading from wp.com helps or not but you're right that they should be served from the site. I've just made that change.